src/IndexingWrapper.cpp

```diff
@@ -98,13 +98,13 @@ parseField(const string & field, const char & separator)
 //' @param dbpath	path to a Xapian database
 //' @param dataFrame	data frame to be indexed
 //' @param idColumn	index of a column in the data frame whose row value
-//will be used as a unique identifier to each document(row)
+//                  will be used as a unique identifier to each document(row)
```

Comment:
This line doesn't align, I suspect because when you're editing it you have different tab settings to the usual. Maybe get rid of the tabs in the rest of this comment block? (eg `@param idColumn` above is followed by a tab.)

Is the `//'` only needed at the start of each `@param` line? I assume it's something to do with Roxygen or similar.